packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -130,6 +133,17 @@ export function executeHandlebarsTemplate(
     cache
   )
 
+  Handlebars.registerHelper('translateDate', function (date: string) {
+    if (!isValid(new Date(date))) {
+      return ''
+    }
+
+    window.__localeId__ = intl.locale
+    return format(new Date(date), 'dd MMMM yyyy', {
+      locale: locales[window.__localeId__]
+    })
+  })
```

Comment:
Should we call `formatLongDate` here?

Reply:
this is not possible in all cases.

formatLongDate expects a date in yyyy-mm-dd format (e.g. "1990-11-11")

but we have registrationDate and certificateDate which are in another format: 19 April 2023. With those, formatLongDate will always return a null value.

That's why I did this.